jwst/ami/matrix_dft.py

```diff
@@ -19,13 +21,14 @@
 calculations on much smaller ones".
 
 Code originally by A. Sivaramakrishnan
-2010-11-05 Revised normalizations for flux conservation consistent
-with Soummer et al. 2007. Updated documentation.  -- M. Perrin
-2011-2012: Various enhancements, detailed history not kept, sorry.
-2012-05-18: module renamed SFT.py -> matrixDFT.py
-2012-09-26: minor big fixes
-2015-01-21: Eliminate redundant code paths, correct parity flip,
-PEP8 formatting pass (except variable names)-- J. Long
+
+* 2010-11-05 Revised normalizations for flux conservation consistent
+  with Soummer et al. 2007. Updated documentation.  -- M. Perrin
+* 2011-2012: Various enhancements, detailed history not kept, sorry.
+* 2012-05-18: module renamed SFT.py -> matrixDFT.py
+* 2012-09-26: minor big fixes
+* 2015-01-21: Eliminate redundant code paths, correct parity flip,
+  PEP8 formatting pass (except variable names)-- J. Long
```

Comment:
This code history is incomplete - I know I made substantial changes to this file last year.  Should we remove it entirely, and just credit the various authors and references?  Or make the additional comment that changes post-201X are logged on GitHub?